main/auth/twitch.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+
+import flask
+
+import auth
+import config
+import model
+import util
+
+from main import app
+
+twitch_config = dict(
+  access_token_method='POST',
+  access_token_url='https://id.twitch.tv/oauth2/token',
+  api_base_url='https://api.twitch.tv/helix/',
+  authorize_url='https://id.twitch.tv/oauth2/authorize',
+  client_id=config.CONFIG_DB.twitch_client_id,
+  client_secret=config.CONFIG_DB.twitch_client_secret,
+  request_token_params={
+    'scope': 'user:read:email',
+    'token_endpoint_auth_method': 'client_secret_post',
+		"Client-ID": config.CONFIG_DB.twitch_client_id,
+  },
+)
+
+twitch = auth.create_oauth_app(twitch_config, 'twitch')
+
+def twitch_compliance_fix(url, headers, data):
+  headers["Client-ID"] = config.CONFIG_DB.twitch_client_id
+  return url, headers, data
+
+twitch.pre_request = twitch_compliance_fix
```

Comment:
what is this `pre_request`? In Authlib, we need to register it as `compliance_fix` like https://github.com/authlib/loginpass/blob/master/loginpass/twitch.py#L54

Reply:
I removed it completely @rbubley.. I think it wasn't called at all.

Reply:
But you still need to add a complaint fix. You can learn it from loginpass.